### ISSUE TITLE
HTML element is depreciated or obsolete

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#3942]
some pages still use html font tag instead of CSS.

This is likely due to copy/pasting of text from a word processor which then specified all the styles.

example

https://www.iastatedigitalpress.com/itaa/article/id/13790/

```html
<font face="Times New Roman, serif">
<span style="font-size: 14.6667px;">  </span>
</font>
```

this should be fixed in the new bleachfield work, going forward, but it runs on the 'save' function, so  when this new feature is added we need to run it across all previously saved text that it will now apply to (e.g. article titles, abstracts).

**Comment by ajrbyers** [comment]
With TinyMCE and BleachField changes this wont be an issue going forward.

https://github.com/BirkbeckCTP/janeway/pull/3983
